plugins/microsoft_teams/plugin.spec.yaml

```diff
@@ -20,7 +20,7 @@ tags:
 - chat
 hub_tags:
   use_cases: [alerting_and_notifications, application_management, threat_detection_and_response, user_management]
-  keywords: [microsoft, teams, office 365, chat]
+  keywords: [microsoft_teams, office365, chat]
```

Comment:
more of a question than a request for a PR change so ultimately approved the PR, but should there be a mention of the update to office365 from 'office 365' to 'office_365' in the help.md?

Reply:
@jrose-r7 Yes - we will change version history to reflect changes :)